### PR TITLE
Add com.dosbox.DOSBox

### PR DESCRIPTION
**Pull request by AsciiWolf**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

**Comment by eszlari** [comment]
The icon that comes with distro packages is this:
https://commons.wikimedia.org/wiki/File:DOSBox_icon.png

My software center (plasma-discover) also displays a appdata text. I would drop references to SDL, I don't think it's relevant for users.

**Comment by AsciiWolf** [comment]
@eszlari Thanks, I have updated the icon. Regarding the AppData file text, it is based on a package description from Debian/Ubuntu and I have already sent the same AppData file to the Fedora DOSBox package maintainer and I don't want the Flathub one to be different than the rest. You are right that it's not relevant for users, but I think it's not confusing for them as well. So I think I will leave this file as it is for now.

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

File: com.dosbox.DOSBox.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                    "path": "dosbox.appdata.xml"
+                },
+                {
+                    "type": "file",
+                    "path": "dosbox.desktop"
+                }
+            ],
+            "post-install": [
+                "mkdir -p /app/share/appdata",
+                "cp dosbox.appdata.xml /app/share/appdata",
```
Comment (TingPing) [inline com.dosbox.DOSBox.json]: `install -Dm644 ...` is a bit cleaner than repeated `mkdir && cp`

File: dosbox.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    DOSBox emulates a 286/386 realmode CPU, Directory FileSystem/XMS/EMS, a SoundBlaster card
+    for excellent sound compatibility with older games... You can "re-live" the good old days
+    with the help of DOSBox, it can run plenty of the old classics that don't run on your
+    new computer!
+    </p>
+    <p>
+    This version of DOSBox is sandboxed. You have to copy your DOS files to your Documents
+    directory and mount them using a mount command in your DOSBox.
+    </p>
+  </description>
```
Comment (TingPing) [inline dosbox.appdata.xml]: Release information and screenshots would be nice.

File: com.dosbox.DOSBox.json (hunk 1 above)
Comment (AsciiWolf) [reply com.dosbox.DOSBox.json]: Done.

File: dosbox.appdata.xml (hunk 2 above)
Comment (AsciiWolf) [reply dosbox.appdata.xml]: Done.

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

**Comment by nedrichards** [comment]
You shouldn't disable aarch64 entirely. You just need to rm-configure and then do a local reconfigure to use tools new enough to realise what 64 bit arm is. There's a bunch of other examples of this on flathub.

**Comment by AsciiWolf** [comment]
bot, build com.dosbox.DOSBox

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.dosbox.DOSBox

**Pull request closed without merging**